integrations/opsgenie/integration.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Literal
+from pydantic import Field, BaseModel
+
+from port_ocean.core.handlers import APIPortAppConfig
+from port_ocean.core.handlers.port_app_config.models import (
+    ResourceConfig,
+    PortAppConfig,
+    Selector,
+)
+from port_ocean.core.integrations.base import BaseIntegration
+
+
+class APIQueryParams(BaseModel):
+    created_at: str | None = Field(
+        alias="createdAt", description="The date and time the alert was created"
```

Comment:
its being used only by the alerts kind, so lets provide it a more specific name

Reply:
This is used in both alerts and incidents